src/PhpWord/Writer/Word2007/Element/ParagraphWordWrap.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of PHPWord - A pure PHP library for reading and writing
+ * word processing documents.
+ *
+ * PHPWord is free software distributed under the terms of the GNU Lesser
+ * General Public License version 3 as published by the Free Software Foundation.
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code. For the full list of
+ * contributors, visit https://github.com/PHPOffice/PHPWord/contributors.
+ *
+ * @see         https://github.com/PHPOffice/PHPWord
+ * @copyright   2010-2018 PHPWord contributors
+ * @license     http://www.gnu.org/licenses/lgpl.txt LGPL version 3
+ */
+
+namespace PhpOffice\PhpWord\Writer\Word2007\Element;
+
+/**
+ * @since 0.17.0
+ */
+class ParagraphWordWrap
```

Comment:
no need for this, WordWrap is just a simple on/off, there are no other attributes possible (see http://www.datypic.com/sc/ooxml/e-w_wordWrap-1.html)

Reply:
Sorry ,my English is not well ,so i have some trouble to talk with you ,but i think you can add the wordwrap future to phpword after you resolve it by yourself .